ui/src/me/components/account/__snapshots__/Tokens.test.tsx.snap

```diff
@@ -56,14 +56,16 @@ exports[`Account rendering renders! 1`] = `
   <ResourceFetcher
     fetcher={[Function]}
   >
-    <SpinnerContainer
+    <t
```

Comment:
Maybe we should turn off uglification in the clockface builds so that we get more descriptive snapshots.

Reply:
Ya, this was something we also discussed with @bthesorceror, we may just.